libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/placeslistonmap/PlacesListOnMapScreen.kt

```diff
@@ -176,8 +181,8 @@ class PlacesListOnMapScreen(
                 )
             }
             val featureCollection = FeatureCollection.fromFeatures(features)
-            carMapSurface.mapSurface.getMapboxMap().getStyle { style ->
-                placesLayerUtil.updatePlacesListOnMapLayer(style, featureCollection)
+            carMapSurface.getStyle {
+                placesLayerUtil.updatePlacesListOnMapLayer(it, featureCollection)
```

Comment:
Will we redraw the points after style change? 
Since this is called on `onGetTemplate` we probably have to call `invlidate` whenever style changes.

Reply:
I've tested it with going to settings, changing a day/night and returning to the app. Favorites screen is closed, so we need to click `Favorites` again to see it. We should keep it separately if we want to restore the screen.

Reply:
retested it with runtime style change from day to night, no issues with pins, no need to add `invalidate()`
the same with a destination pin